src/Login/Login.css

```diff
@@ -0,0 +1,20 @@
+main {
+  justify-content: center;
+  min-height: 80vh;
+}
+
+form {
+  align-self: center;
+  flex-direction: column;
+  min-height: 20vh;
+}
```

Comment:
awesome! we should try to find a picture of a book so that we can use this for the login form, give it a little stylistic touch